solana/program_v2/packages/tests/src/gatekeeper-suite/update-gatekeeper.test.ts

```diff
@@ -1,9 +1,9 @@
-import { AdminService } from '@identity.com/gateway_v2-client/src/AdminService';
-import { NetworkService } from '@identity.com/gateway_v2-client/src/NetworkService';
-import { GatewayV2 } from '@identity.com/gateway_v2-idl/src/gateway_v2';
+import { AdminService } from '@identity.com/gateway-solana-client';
```

Comment:
we can merge these like:
`import { AdminService, NetworkService } from '@identity.com/gateway-solana-client';`

Reply:
Oh shoot missed this